backend/app/cmd/server.go

```diff
@@ -454,25 +454,14 @@ func (s *ServerCommand) newServerApp(ctx context.Context) (*serverApp, error) {
 		return nil, errors.Wrap(err, "failed to make avatar store")
 	}
 	authRefreshCache := newAuthRefreshCache()
-	authenticator, err := s.makeAuthenticator(ctx, dataService, avatarStore, adminStore, authRefreshCache)
+	authenticator := s.makeAuthenticator(dataService, avatarStore, adminStore, authRefreshCache)
+
+	telegramAuth, err := s.addAuthProviders(authenticator)
```

Comment:
this return value seems very unexpected for the used-to-be generic call. I.e. getting tg-specific TGUpdatesReceiver from addAuthProviders doesn't fill right to me. I would rather split it and will have addTgAuthProvider as a separate func returning tg-related updater